app/src/main/java/com/spendster/presentation/homeScreen/ProfileFragment.java

```diff
@@ -1,23 +1,40 @@
 package com.spendster.presentation.homeScreen;
 
 import android.os.Bundle;
+import android.support.annotation.NonNull;
 import android.support.v4.app.Fragment;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
+import android.widget.EditText;
 
 import com.spendster.R;
+import com.spendster.data.entity.User;
+import com.spendster.presentation.authentication.SharedPreferencesUserStorage;
 
 public class ProfileFragment extends Fragment {
 
+    private EditText etUsername, etProfileEmail;
+
     public static ProfileFragment newInstance(){
-        ProfileFragment profileFragment = new ProfileFragment();
-        return profileFragment;
+        return new ProfileFragment();
     }
 
     @Override
-    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+    public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
-        return inflater.inflate(R.layout.profile_fragment, null);
+        View view = inflater.inflate(R.layout.profile_fragment, container, false);
+        etUsername = view.findViewById(R.id.etUsername);
+        etProfileEmail = view.findViewById(R.id.etProfileEmail);
+        initUI();
+        return view;
+    }
+
+    private void initUI(){
+        SharedPreferencesUserStorage sharedPreferencesUserStorage = new
```

Comment:
@SimonMaha this should be done in presenter. Activities as other views should only init ui with direct data